extension.js

```diff
@@ -128,7 +128,7 @@ function destroyOverlay() {
 function onHotkeyPressed() {
   Log.debug("Hot key pressed");
   //Clear the lock bit so the status of Caps_Lock won't affect the functionality
-  let multiKeysCode = keymap.get_modifier_state() & (~2);
+  let multiKeysCode = keymap.get_modifier_state() & (~(2 | 16));
```

Comment:
I nearly forgot how mask works.. I think 2 is capslock and 16 is NumsLock , so doing a "bitwise or" will mask both 2 and 16?

Reply:
Ok, I will test it.

Reply:
![image](https://user-images.githubusercontent.com/22115961/172070741-36f4a614-25d1-44e0-a0ca-fb16c183f104.png)
@pbxqdown I've tested this suggestion. It works.

Reply:
P.S. But there is one point. When I changed the code and restarted gnome-shell `Alt + F2 -> r`. Code 8 appeared in the logs when the Alt key was pressed. However, the old number 24 remained in the extension settings window, which I had to change manually to 8.

Reply:
Nice catch! This line needs to change as well: https://github.com/pbxqdown/gnome-shell-extension-transparent-window/blob/e4261bd9b47adcb9863c7ef11b6e46a89a33eaa4/prefs.js#L48

Reply:
Will be better to move these two places to one function in utils.js.
For example, to create a function 
```js
// utils.js
function getMultiKeysCode() {
    keymap.get_modifier_state() & (~(2 | 16));
}
```
and then import it into two places, as isVersionGreaterOrEqual() for example.

Reply:
@pbxqdown let's wait until the weekend. If it will not be done, I can do it.

Reply:
@igolskyi Sure, thanks for helping with this!

Reply:
@pbxqdown I've done it here https://github.com/pbxqdown/gnome-shell-extension-transparent-window/pull/20. Please review it.